modernomad/settings/common.py

```diff
@@ -94,8 +94,14 @@
     'django.contrib.auth.backends.ModelBackend'
 )
 
-EMAIL_BACKEND = 'modernomad.backends.MailgunBackend'
 MAILGUN_API_KEY = env('MAILGUN_API_KEY', default='')
+if MAILGUN_API_KEY:
+    EMAIL_BACKEND = 'modernomad.backends.MailgunBackend'
+    # This should only ever be true in the production environment. Defaults to False.
+    MAILGUN_CAUTION_SEND_REAL_MAIL = env('MAILGUN_CAUTION_SEND_REAL_MAIL', default='')
+else:
+    EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
```

Comment:
I'd say remove this and keep the console backend in settings local. Right now we're not using an email backend at all. You're only using an email backend when you use the `send_mail` function from `from django.core.mail import send_mail`. 

Using the email backend with `send_mail` would mean it'd be impossible to send real email when using local even though you had the correct `MAILGUN_API_KEY` configured.

Reply:
Sorry I'm being dense this morning! :smile: I don't quite follow this point about right now we're not using an email backend?

Reply:
So right now we're using this function. https://github.com/embassynetwork/modernomad/blob/master/core/emails.py#L38 to send out email, right? (A function that also happened to be duplicated as you noticed). 

The email backend could also be used to send out email. Ie the one defined here -> https://github.com/embassynetwork/modernomad/blob/master/modernomad/backends.py#L18 Personally I wouldn't trust it too much since it hasn't been used at all and has no tests. 

For an EmailBackend to be used, you need to call the django function `send_mail`, that function uses any backend that you have defined in settings.py. Since right now, we're not using `send_email` anywhere in the project the email backend isn't used at all. 

So, instead of rolling things our own, I think it'd be wiser to rely on a library such as anymail.

Reply:
You can read a bit more about the email backends here: https://docs.djangoproject.com/en/2.1/topics/email/#email-backends

Reply:
I see thanks @jonathan-s :). I was confused because we _are_ using the django email backend, but only to send to the console (or at least we were... unless we broke something which is entirely possible :joy:). But I have not found a need for something other than console + mailgun, personally. 

@bfirsh re putting the settings in common, yeah that's where i have it in this PR, for that very reason :).